### ISSUE TITLE
Training Loss

### ISSUE THREAD

**Issue by MuhammadAsadJaved** [YunYang1994/tensorflow-yolov3#568]

**Comment by MuhammadAsadJaved** [comment]
Hi, 
I am training YOLOV3  with 1 class and dataset is about 7000 images.

The training loss decrease for few iterations and then stop decreasing, test loss is always NAN.  
I am using default settings and anchors 
here are my configurations and loss  reports. 
What is the possible reasons and how to overcome this problem? 
![config](https://user-images.githubusercontent.com/28862708/89607583-760d2d80-d8a5-11ea-8ef1-680a30ed61cb.png)
![loss](https://user-images.githubusercontent.com/28862708/89607590-786f8780-d8a5-11ea-8c23-d6504f887f52.png)

**Comment by MrZhousf** [comment]
change np.mean to np.nanmean
like this:
train_epoch_loss, test_epoch_loss = np.nanmean(train_epoch_loss), np.nanmean(test_epoch_loss)

**Comment by MuhammadAsadJaved** [comment]
@MrZhousf  OK. Thank you so much. I'll give it a try.
